livy/src/main/java/org/apache/zeppelin/livy/LivyVersion.java

```diff
@@ -0,0 +1,96 @@
+package org.apache.zeppelin.livy;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Provide reading comparing capability of livy version
+ */
+public class LivyVersion {
+  private static final Logger logger = LoggerFactory.getLogger(LivyVersion.class);
+
+  private static final LivyVersion LIVY_0_2_0 = LivyVersion.fromVersionString("0.2.0");
```

Comment:
Not tested yet, I think the first version of livy interpreter is based on livy 0.2